ODIN_II/regression_test/benchmark/task/syntax/simulation_result.csv

```diff
@@ -1,120 +1,132 @@
+-1                , 1            , 176          , 4.8         , 3.3                , n/a                                   , 64   , -1    , -1   , 6           , 176           , 93.3            , 0   , 13.8         , constant_module_inst.blif                                  , syntax/constant_module_inst/no_arch                                                             , -1     , 4  , -1        , 12          , 241       , 4  
+4                 , 0            , 8            , 13.3        , 27.7               , k6_frac_N10_frac_chain_mem32K_40nm.xml, 0    , 4     , -1   , 4           , 8             , 100.0           , 0   , 54.0         , constant_module_inst.blif                                  , syntax/constant_module_inst/k6_frac_N10_frac_chain_mem32K_40nm                                  , -1     , 4  , -1        , 8           , 12        , 5  
+4                 , 1            , 160          , 175.8       , 651.4              , k6_frac_N10_frac_chain_mem32K_40nm.xml, 128  , 96    , -1   , 4           , 160           , 100.0           , 0   , 702.1        , inferred_ram_w_clog2.blif                                  , syntax/inferred_ram_w_clog2/k6_frac_N10_frac_chain_mem32K_40nm                                  , -1     , 160, -1        , 5           , 385       , 37 
+4                 , 1            , 20           , 5.6         , 43.1               , k6_frac_N10_frac_chain_mem32K_40nm.xml, 8    , -1    , -1   , 5           , 20            , 100.0           , 0   , 66.8         , h7_of_8_bit_for_pass_through.blif                          , syntax/h7_of_8_bit_for_pass_through/k6_frac_N10_frac_chain_mem32K_40nm                          , -1     , 8  , -1        , 7           , 29        , 8  
+-1                , 0            , 2            , 4.3         , 0.3                , n/a                                   , 0    , -1    , -1   , 4           , 2             , 66.7            , 0   , 1.7          , timescale_syntax.blif                                      , syntax/timescale_syntax/no_arch                                                                 , -1     , 1  , -1        , 4           , 2         , 2  
+-1                , 1            , 14           , 4.3         , 0.4                , n/a                                   , 4    , -1    , -1   , 5           , 14            , 86.7            , 0   , 2.3          , bm_DL_4_bit_updown_counter.blif                            , syntax/bm_DL_4_bit_updown_counter/no_arch                                                       , -1     , 2  , -1        , 15          , 19        , 5  
```

Comment:
Can you only update the line that really changed? Also there seems to be a bug on the python script since the order is way off. what platform are you using ? I will have to test and make sure ordering is preserved...

Reply:
I am using Ubuntu 16.04

Reply:
pr #1245 should fix this. once merge can you regenerate the file?

Reply:
@jeanlego Sure I will do it.